node/src/service.rs

```diff
@@ -13,6 +13,10 @@ use sc_transaction_pool_api::OffchainTransactionPoolFactory;
 use sp_consensus_aura::sr25519::AuthorityPair as AuraPair;
 use std::{sync::Arc, time::Duration};
 
+/// The minimum period of blocks on which justifications will be
+/// imported and generated.
+const GRANDPA_JUSTIFICATION_PERIOD: u32 = 512;
```

Comment:
How do we arrive at this value ? Is 512 best practice , or is it implict in our existing code base and exposed in v1.10?

Reply:
It was the period used previously, it was just hardcoded rather than in a variable. It was moved to a variable because it's repeated in a couple places now.